api/src/mailbox.rs

```diff
@@ -37,18 +37,28 @@ pub struct MessageToDelete
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
 pub struct MessageRequest
 {
+    pub sender: Option<Sender>,
     pub action: String,
     pub uuid: String,
     pub data: Payload,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+pub struct Sender
+{
+    pub signing_key: String,
+    pub encryption_key: String,
+    pub cid: String,
```

Comment:
do these need to be marked as optional? I'm not familiar with rust schemas

Reply:
Yeah sry